scripts/style-spec.js

```diff
@@ -1 +1,4 @@
-var spec = module.exports = require('../mapbox-gl-js/src/style-spec/reference/v8');
+var spec = module.exports = require('../vendor/mapbox-gl-native/mapbox-gl-js/src/style-spec/reference/v8');
+
+// FIXME: https://github.com/mapbox/mapbox-gl-native/issues/15008
+delete spec.layout_circle["circle-sort-key"];
```

Comment:
Ideally, gl-native would also centralize style specification overrides in scripts/style-spec.js so that this repository could simply require the file from that submodule instead of maintaining a separate copy.